examples/llama2/llama2_pipeline_parallel.json

```diff
@@ -0,0 +1,47 @@
+{
+    "input_model":{
+        "type": "PyTorchModel",
+        "config": {
+            "model_script": "user_script.py",
+            "io_config": "get_merged_decoder_with_past_io_config",
+            "dummy_inputs_func": "get_merged_decoder_with_past_dummy_inputs",
+            "hf_config": {
+                "model_name": "meta-llama/Llama-2-7b-hf",
+                "model_class": "LlamaForCausalLM",
+                "task": "text-generation"
+            }
+        }
+    },
+    "systems": {
+        "local_system": {
+            "type": "LocalSystem",
+            "config": {
+                "accelerators": ["gpu"]
+            }
+        }
+    },
+    "passes": {
+        "pipeline_parallel": {
+            "type": "PyTorchPipelineParallel",
+            "config": {
+                "user_script": "llama2_pipeline_parallel.py",
```

Comment:
This was copy/paste error from llama2_tensor_parallel.json example. The field value is unused since the layers implementation is part of Olive and not coming from examples folder. I have fixed both example configuration so there are no dangling references but the fields itself remain unused.